CONTRIBUTING.md

```diff
@@ -48,7 +48,9 @@ top of things.
 
 * Make sure you have added the necessary tests for your changes.
 * Run _all_ the tests to assure nothing else was accidentally broken.
-  * We recommend running `./check.sh` to have the same checks run as on github
+* We recommend running the follwing rake tasks:
+  * `bundle exec rake spec_random` - runs unit tests
+  * `bundle exe rake spec_integration` runs integration tests. Note that tests with the `skip_outside_ci` tag are excluded outside of CI, you can run them with `CI=true bundle exe rake spec_integration`
```

Comment:
Maybe we should also recommend running `bundle exec rake rubocop` and `bundle exec rake check` as a 3 in 1 solution?